### PR TITLE
Applied fixes from StyleCI

### PR DESCRIPTION
**Pull request by GrahamCampbell**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://styleci.io).

---

For more information, click [here](https://styleci.io/analyses/XZ9G68).

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Hi there. I've noticed your repo was failing our analyses. You also have two actual syntax errors in your files that we're picking up that prevent the analysis from "passing".

**Comment by GrahamCampbell** [comment]
Ahh, right. I see that was intentional. You need to update your config to exclude those files then.

**Comment by GrahamCampbell** [comment]
Ping me if you need a hand.

**Comment by ovr** [comment]
@GrahamCampbell 
Thanks for your attention ;)

**Comment by GrahamCampbell** [comment]
> Why this drop @package annotation?

Package annotations are meaningless and shouldn't be used. Their only use case was in php 5.2 before namespaces.

File: tests/PHPSA/Expression/BinaryOp/EqualTest.php [hunk 1]
```diff
@@ -1,14 +1,12 @@
 <?php
 
-namespace Tests\PHPSA\Expression\BinaryOp;
+namespace tests\PHPSA\Expression\BinaryOp;
```
Comment (GrahamCampbell) [inline tests/PHPSA/Expression/BinaryOp/EqualTest.php]: Looks like the psr0 fixer needs turning off.

**Comment by GrahamCampbell** [comment]
This PR can't be merged until the styleci config is corrected.

**Comment by GrahamCampbell** [comment]
You need to disable the psr0 fixer and exlude the test stubs.

**Pull request closed without merging**